wasm.md

```diff
@@ -1531,6 +1496,238 @@ Start Function
+    syntax GlobalSpec ::= InlineExport GlobalSpec
+ // ---------------------------------------------
+    rule <k> ( global                  EXPO:InlineExport SPEC:GlobalSpec )
+          => ( global #freshId(NEXTID) EXPO              SPEC            )
+          ...
+         </k>
+         <nextFreshId> NEXTID => NEXTID +Int 1 </nextFreshId>
+
+    rule <k> ( global ID:Identifier ( export ENAME ) SPEC:GlobalSpec )
+          => ( export ENAME ( global ID ) )
+          ~> ( global ID SPEC )
+          ...
+         </k>
+
+    syntax FuncSpec   ::= InlineExport FuncSpec
+ // -------------------------------------------
+    rule <k> ( func                  EXPO:InlineExport SPEC:FuncSpec )
+          => ( func #freshId(NEXTID) EXPO              SPEC          )
+          ...
+         </k>
+         <nextFreshId> NEXTID => NEXTID +Int 1 </nextFreshId>
+
+    rule <k> ( func ID:Identifier ( export ENAME ) SPEC:FuncSpec )
+          => ( export ENAME ( func ID ) )
+          ~> ( func ID SPEC )
+          ...
+         </k>
+
+    syntax TableSpec  ::= InlineExport TableSpec
+ // --------------------------------------------
+    rule <k> ( table                  EXPO:InlineExport SPEC:TableSpec )
+          => ( table #freshId(NEXTID) EXPO              SPEC           )
+          ...
+         </k>
+         <nextFreshId> NEXTID => NEXTID +Int 1 </nextFreshId>
+
+    rule <k> ( table ID:Identifier ( export ENAME ) SPEC:TableSpec )
+          => ( export ENAME ( table ID ) )
+          ~> ( table ID SPEC )
+          ...
+         </k>
+
+    syntax MemorySpec ::= InlineExport MemorySpec
+ // ---------------------------------------------
+    rule <k> ( memory                  EXPO:InlineExport SPEC:MemorySpec )
+          => ( memory #freshId(NEXTID) EXPO              SPEC            )
+          ...
+         </k>
+         <nextFreshId> NEXTID => NEXTID +Int 1 </nextFreshId>
+
+    rule <k> ( memory ID:Identifier ( export ENAME ) SPEC:MemorySpec )
+          => ( export ENAME ( memory ID ) )
+          ~> ( memory ID SPEC )
+          ...
+         </k>
+```
+
+Imports
+-------
+
+Imports need to describe the type of what is imported.
+That an import is really a subtype of the declared import needs to be checked at instantiation time.
+The value of a global gets copied when it is imported.
+
+```k
+    syntax Defn       ::= ImportDefn
+    syntax ImportDefn ::= "(" "import" String String ImportDesc ")"
+    syntax ImportDesc ::= "(" "func"   OptionalId TypeUse        ")" [klabel(funcImportDesc)]
+                        | "(" "table"  OptionalId TableType      ")" [klabel( tabImportDesc)]
+                        | "(" "memory" OptionalId MemType        ")" [klabel( memImportDesc)]
+                        | "(" "global" OptionalId TextGlobalType ")" [klabel(globImportDesc)]
+ // -------------------------------------------------------------------------------------
+    rule <k> ( import MOD NAME (func OID:OptionalId TUSE:TypeUse)) => . ... </k>
```

Comment:
you have space after first paren but not before the last paren.

Reply:
Fixed.